src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -314,10 +314,18 @@
                                     :binding "mod+shift+p"
                                     :fn      (fn [] (state/toggle! :ui/command-palette-open?))}
 
-   :select-graph/open              {:desc    "Open select graph component"
```

Comment:
Want to make the namespace consistent and choose a more feature oriented namespace. The previous namespace wasn't a good choice as `:select-graph` is an implementation detail